src/App.js

```diff
@@ -1,28 +1,20 @@
-import logo from './logo.svg';
+import { Routes, Route } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import NavBar from './component/Nav/NavBar';
+import HomePage from './component/Home/HomePage';
+import CountryDetail from './component/Country/aCountry/singleCountryInfo';
+import store from './app/store';
 import './App.css';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Provider store={store}>
+      <Routes>
+        <Route path="/" element={<NavBar />} />
+        <Route index element={<HomePage />} />
+        <Route path=":code" element={<CountryDetail />} />
```

Comment:
- [ ] Kindly adhere to the project requirement, your previous code reviewer also mentioned it, even though you did a great job by making the changes still you need to make your design look as close as possible to the `Behance` design. You can even compare your design in the below given table, can easily figure out that you are missing a few things that need to be added to pass this capstone project. Implementing the things as per the given design will make you more better and detail-oriented developer.

### Requirement
![image](https://user-images.githubusercontent.com/103386183/268376852-0e7566b5-be90-4885-befd-e429503c5314.png)

### Home Design Comparison
<table>
<tr>
<th>Behance Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-04 14-58-52](https://user-images.githubusercontent.com/103386183/268375466-89aa2857-1ba6-472c-8ceb-aa156ab7fd06.png)

</td>
<td>

![Screenshot from 2023-09-16 18-23-01](https://github.com/faisalakb/React-Redux-Capstone/assets/66476329/cc1f5d31-b11d-4584-9a01-6d5f6858536e)


</td>
</tr>
</table>

### Detail Design Comparison

<table>
<tr>
<th>Behance Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-04 14-58-52](https://user-images.githubusercontent.com/103386183/268376542-23147907-83ed-497f-8398-c44a07f56f0a.png)

</td>
<td>

![Screenshot from 2023-09-16 18-30-01](https://github.com/faisalakb/React-Redux-Capstone/assets/66476329/e469bdd8-ced2-4680-b003-21a6a8509165)


</td>
</tr>
</table>